packages/itwinui-css/src/button/button.scss

```diff
@@ -37,6 +37,10 @@
   transition: fill var(--iui-duration-1) ease-out;
   fill: currentColor;
   margin-block: var(--iui-size-2xs);
+
+  > * {
+    display: flex;
+  }
```

Comment:
Jon suggested this idea of using `display: flex` so that the `<svg-document>` inside the wrapper `<span>` will expand to the size of the wrapper `<span>`. Without it, if the `<span>` for example was `24x24`, then the `<svg-document>` would be `0x24`.

This works, but I was just wondering, why does it work? Also for e.g. in React, we directly have `<svg>` instead of `<svg-document>`. There it works without L41-43. Why is that so?

CSS
![image](https://user-images.githubusercontent.com/45748283/193146085-ac348eed-d84e-4fee-a4c5-a1f273dacbb7.png)

React
![Screenshot from 2022-09-29 13-38-32](https://user-images.githubusercontent.com/45748283/193145967-af482123-0b6b-4e10-a882-a82b629aa0d3.png)

Reply:
Typically `> *` has me pretty nervous because we're never sure what might be nested, but because this is being applied on `.iui-button-icon` I'm thinking it might be ok here.